crm_claim_rma/models/account_invoice_line.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
The key in the guidelines is the word "main_model", that tells about what I'm saying. I remember to have more explicit instructions about it, but now I can't find it. We can make an amendment in the guidelines to say it more explicitly